### PR TITLE
docs: add first draft of LSP11 specs

### PR DESCRIPTION
**Pull request by YamenMerhi**
## What does this PR introduce?
- Introducing **LSP11BasicSocialRecovery** standard specs

### CODE REVIEW

File: LSPs/LSP-11-BasicSocialRecovery.md [hunk 1]
```diff
@@ -0,0 +1,267 @@
...
+type: LSP
+created: 2021-2-14
+requires: ERC725, LSP0, LSP2, LSP6
+---
+
+## Simple Summary
+This standard describes a **basic social recovery** contract that can recover access to [ERC725](https://github.com/ERC725Alliance/ERC725/blob/develop/docs/ERC-725.md) contracts through the [LSP6-KeyManager](./LSP-6-KeyManager.md).
+
+## Abstract
+This standard allows for recovering access to ERC725 contracts such as tokens, NFTs, and Universal Profiles by adding a new address to control them through the Key Manager after a recovery process.
```
Comment (CallumGrindle) [inline LSPs/LSP-11-BasicSocialRecovery.md:16]:
```suggestion
This standard provides a mechanism for recovering access to ERC725 contracts such as tokens, NFTs, and Universal Profiles by adding a new address to control them through the Key Manager after a recovery process.
```

File: LSPs/LSP-11-BasicSocialRecovery.md [hunk 2]
```diff
@@ -0,0 +1,267 @@
...
+
+The social recovery contract should provide the most flexible and secure process, where the guardians, set originally by the owner, can vote to one address in each recoverProcessId. The address willing to recover can choose a recoverProcessId where he reached the guardiansThreshold and after successfully providing the right secret word, that produces the hash already set by the owner, he will be granted the owner permissions in the account recovered and a new hash will be set and all previous recoverProcessId will be invalidated.
```
Comment (CallumGrindle) [inline LSPs/LSP-11-BasicSocialRecovery.md:18]:
```suggestion
The social recovery contract should ensure a flexible and secure process where guardians, nominated by the owner, can vote for one address in each `recoverProcessId`. The address requesting recovery can choose a `recoverProcessId` where he reached the `guardiansThreshold` and after successfully providing the right secret word which produces the same hash as the secret word set by the owner, he will be granted the owner permissions in the recovered account and a new hash will be set and all previous `recoverProcessId` will be invalidated.
```

I tried to make this a bit more clear but please double check the content is still correct. 

I think this passage could do with a more clear bullet pointed process so its clear what happens step by step

File: LSPs/LSP-11-BasicSocialRecovery.md [hunk 3]
```diff
@@ -0,0 +1,267 @@
...
+
+## Motivation
+Any Key could be lost or leaked due to a certain accident, so it's not advised to rely on one singular key to control ERC725 contracts through the Key Manager and a social recovery contract is needed in this case.
+
+In the case above, the user can simply reach out to his guardians and ask them to vote for a certain address. 
+There are many possible choices for whom to select as a guardian. The three most common choices are:
+
+- Other devices (or paper mnemonics) owned by the wallet holder themselves
```
Comment (CallumGrindle) [inline LSPs/LSP-11-BasicSocialRecovery.md:26]:
```suggestion
- Other EOAs controlled by the wallet holder themselves (via paper mnemonics or cold storage wallets) 
```

File: LSPs/LSP-11-BasicSocialRecovery.md [hunk 4]
```diff
@@ -0,0 +1,267 @@
...
+- Friends and family members (EOAs or Universal Profiles)
+- Institutions, which would vote for the address provided if they get a confirmation of your phone number or email or perhaps in high-value cases to verify you personally by video call
```
Comment (CallumGrindle) [inline LSPs/LSP-11-BasicSocialRecovery.md:28]:
```suggestion
- Institutions, which could vote for a provided address if they get a valid confirmation via phone number, email or video call
```

File: LSPs/LSP-11-BasicSocialRecovery.md [hunk 5]
```diff
@@ -0,0 +1,267 @@
...
+}
+```
+
+### Events
+
+## Rationale
+
+This standard was inspired by the current recovery process in some crypto wallets but this recovery process is a balance between a secret hash and guardians.
+
+In this case, you can ensure that you're guardians can't act maliciously and would need a secret word to recover. The same goes for the secret word if it's exposed, only addresses who reached the guardiansThreshold can recover using it.
```
Comment (CallumGrindle) [inline LSPs/LSP-11-BasicSocialRecovery.md:226]:
```suggestion
In this case, it is ensured that guardians can't act maliciously and would need a secret word to recover. The same goes for the secret word if it is exposed, only addresses who reached the guardiansThreshold can use it to recover an account.
```

**Review by CallumGrindle — COMMENTED** [review]
Cool, I left a few suggestions. One thing that might be worth clarifying, does this standard only apply to ERC725 contracts which implement LSP6 and have a key manager already set up? As this is a prerequisite for implementing this standard

**Comment by YamenMerhi** [comment]
@CallumGrindle Thanks for reviewing! very helpful.
Regarding your question, it's already mentioned in the Simple Summary that this standard recover ERC725 contracts through LSP6-KeyManager.
> This standard describes a basic social recovery contract that can recover access to ERC725 contracts through the LSP6-KeyManager

Do you think we should clarify more? If yes do you have any suggestion?

**Review by b00ste — APPROVED** [review]

**Review by skimaharvey — APPROVED** [review]

File: LSPs/LSP-11-BasicSocialRecovery.md [hunk 6]
```diff
@@ -0,0 +1,345 @@
...
+discussions-to: https://discord.gg/E2rJPP4
+status: Draft
+type: LSP
+created: 2021-2-14
+requires: ERC165, ERC725, LSP2, LSP6
+---
+
+## Simple Summary
+
+This standard describes a **basic social recovery** contract that can recover access to [ERC725] contracts through the [LSP6-KeyManager](./LSP-6-KeyManager.md).
```
Comment (CJ42) [inline LSPs/LSP-11-BasicSocialRecovery.md:14]:
```suggestion
This standard describes a **basic social recovery** contract that can recover access to [ERC725] contracts through a [LSP6-KeyManager](./LSP-6-KeyManager.md).
```

File: LSPs/LSP-11-BasicSocialRecovery.md [hunk 7]
```diff
@@ -0,0 +1,345 @@
...
+
+## Abstract
+
+This standard provides a mechanism for recovering access to ERC725 contracts such as tokens, NFTs, and Universal Profiles by adding a new controller address through the Key Manager after a recovery process.
+
+The social recovery contract should ensure a flexible and secure process where guardians, nominated by the owner, can select one address. The address after reaching the `guardiansThreshold` and after successfully providing the secret word which produces the same hash as the secret hash set by the owner, will be granted the owner permissions in the recovered target and a new hash will be set and a new recovery counter will be created.
+
+## Motivation
+
+Any Key could be lost or leaked due to a certain accident, so it's not advised to rely on one singular key to control ERC725 contracts through the Key Manager and a social recovery contract is needed in this case.
```
Comment (CJ42) [inline LSPs/LSP-11-BasicSocialRecovery.md:24]:
```suggestion
Any Key could be lost or leaked due to a certain accident, so it is not advised to rely on a singular key to control ERC725 contracts through the Key Manager and a social recovery contract is needed in this case.
```

File: LSPs/LSP-11-BasicSocialRecovery.md [hunk 8]
```diff
@@ -0,0 +1,345 @@
...
+
+In the case above, the user can simply reach out to his guardians and ask them to vote for a certain address.
```
Comment (CJ42) [inline LSPs/LSP-11-BasicSocialRecovery.md:26]:
```suggestion
In the case above, the user can reach out to his guardians and ask them to vote for a specific address. 
```

File: LSPs/LSP-11-BasicSocialRecovery.md [hunk 9]
```diff
@@ -0,0 +1,345 @@
...
+This standard provides a mechanism for recovering access to ERC725 contracts such as tokens, NFTs, and Universal Profiles by adding a new controller address through the Key Manager after a recovery process.
+
+The social recovery contract should ensure a flexible and secure process where guardians, nominated by the owner, can select one address. The address after reaching the `guardiansThreshold` and after successfully providing the secret word which produces the same hash as the secret hash set by the owner, will be granted the owner permissions in the recovered target and a new hash will be set and a new recovery counter will be created.
+
+## Motivation
+
+Any Key could be lost or leaked due to a certain accident, so it's not advised to rely on one singular key to control ERC725 contracts through the Key Manager and a social recovery contract is needed in this case.
+
+In the case above, the user can simply reach out to his guardians and ask them to vote for a certain address. 
+There are many possible choices for whom to select as a guardian. The three most common choices are:
```
Comment (CJ42) [inline LSPs/LSP-11-BasicSocialRecovery.md:27]:
```suggestion
There are many possible options for whom to select as a guardian. The three most common options are:
```

**Review by CJ42 — APPROVED** [review]

**Review by frozeman — APPROVED** [review]

**Pull request merged**